lib/chrome.js

```diff
@@ -440,17 +440,26 @@ var ChromeObject = function() {
                         .send()
                         .responseBody
                     ;
+                    
+                    // response if error
+                    if ("result" in response) {
+                        (function(r) {
+                            if ("subtype" in r && r.subtype == "error") {
+                                console.warn("[WebBrowser]", r.description);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Log Injection occurs when untrusted user input is directly written to log files without proper sanitization. This can allow attackers to manipulate log entries, potentially leading to security issues like log forging or cross-site scripting. To prevent this, always sanitize user input using `encodeURIComponent()` or `DOMPurify.sanitize()` before logging.  Learn more - https://cwe.mitre.org/data/definitions/117.html